### DISCUSSION TITLE
Installation on HPC

### DISCUSSION THREAD

**Discussion by superstar54** [beautiful-atoms/beautiful-atoms#186] [General]
I got two issues when installing Batoms on HPC.

- conda can not install `mamba`

Solution: add `--no-mamba` option.

- scipy can not find ``GLIBCXX_3.4.30' `

It seems that `scipy` installed by conda has a problem.  please re-install scipy using pip
```python
conda activate beautiful_atoms
pip uninstall scipy
pip install scipy
```

**Comment by alchem0x2A** [comment]
Regarding mamba, yes it should usually be installed into the base env. If that's an env user don't have write access, --no-mamba should be passed. (mamba is just way faster than normal conda)

the missing glibcxx issue seems to be a fresh one on scipy https://stackoverflow.com/questions/73317676/importerror-usr-lib-aarch64-linux-gnu-libstdc-so-6-version-glibcxx-3-4-30 caused by conda's library distribution (requires to install explicitly gcc12 for the dynlink).
